js/chatbot/shared/ChatBot.svelte

```diff
@@ -178,6 +180,18 @@
 	}
 
 	$: groupedMessages = value && group_messages(value);
+	$: if (value !== null && value.length > 0) suggestionsVisible = false;
```

Comment:
I believe `$: suggestionsVisible = !(value !== null && value.length > 0)` would be better as it would automatically be set to True if someone clears out the chat history with the new clear icon being added in #9201